languages/ruby/lib/oso/polar/ffi/polar.rb

```diff
@@ -62,6 +66,8 @@ def new_query_from_str(str)
           query = Rust.new_query_from_str(self, str, 0)
           raise FFI::Error.get if query.null?
 
+          process_messages
```

Comment:
shouldn't this go above the `raise FFI::Error.get if query.null?`?

Reply:
👍